vars/kolaTestIso.groovy

```diff
@@ -43,9 +43,9 @@ def call(params = [:]) {
         testIsoRuns1["metal4k"] = {
             try {
                 if (params['scenarios4k']) {
-                    shwrap("cd ${cosaDir} &&  kola testiso -S --qemu-native-4k ${extraArgs4k} --scenarios ${scenarios4k} --output-dir tmp/kola-testiso-metal4k")
+                    shwrap("cd ${cosaDir} &&  kola testiso -S --qemu-native-4k --qemu-multipath ${extraArgs4k} --scenarios ${scenarios4k} --output-dir tmp/kola-testiso-metal4k")
                 } else {
-                    shwrap("cd ${cosaDir} &&  kola testiso -S --qemu-native-4k ${extraArgs4k} --output-dir tmp/kola-testiso-metal4k")
+                    shwrap("cd ${cosaDir} &&  kola testiso -S --qemu-native-4k --qemu-multipath ${extraArgs4k} --output-dir tmp/kola-testiso-metal4k")
```

Comment:
What's the background for this change? We're already testing multipath lower down. If we want to add coverage for multipath *and* metal4k together, then let's add a new invocation in the block below instead of modifying this one. That way it also gets gated by the `skipMultipath` variable.

Otherwise, we're not getting any vanilla metal4k coverage.

Reply:
All tests for 4k in FCOS and RHCOS that I checked are testing using `--qemu-multipath` as [here](https://github.com/coreos/fedora-coreos-pipeline/blob/main/jobs/build.Jenkinsfile#L479) . I also assumed FCOS/RHCOS were the only ones using the lib

Reply:
Ahh gotcha. Yeah, ideally we'd have the pipeline use `kolaTestIso` in the future to share this.